docs/reference/6-data-import/getting-started/installation/docker-deployment-for-data-import/readme.md

```diff
@@ -0,0 +1,31 @@
+# Docker Deployment for Data Import
+
+## Overview 
+
+The Docker Deployment for Data Import provides deployment scripts to install Data Import within a Docker-based environment.
+
+## Pre-requisites
+
+To run Docker Deployment for Data Import, you will need:
+
+*   Access to a Docker environment.  The instructions below use [Docker Desktop](https://www.docker.com/products/docker-desktop/) for workstation usage.  [Docker](https://www.docker.com/) is also available for on-premise and cloud providers as well.
+*   An execution shell, such as PowerShell or bash using WSL, Mac OS or Linux.
+
+## Using Docker Deployment for Data Import
+
+### Docker within Non-Developer Use Cases (Production, Staging, etc.) 
+
+Images for Data Import v2.3.2 are available at: [https://hub.docker.com/r/edfialliance/data-import](https://hub.docker.com/r/edfialliance/data-import).
+
+### Reference
+
+Please refer [Docker Deployment - Ed-Fi Tools - Ed-Fi Tech Docs](https://edfi.atlassian.net/wiki/display/EDFITOOLS/Docker+Deployment) for more details.
+
+### Feedback
+
+If you have any feedback to improve this release - suggestions, bug reports, log files for support cases, new feature requests, unmet use cases or other items [\-](https://golflefleur.com/products/glitter) please register a ticket on Ed-Fi Tracker.
```

Comment:
What happened here? Glitter? 

Anyway, here you can see one of the benefits of this documentation migration - not only the weird URL, but noticing the reference to opening a ticket on Ed-Fi Tracker. We need to replace that with:

```
please open a case in the [Ed-Fi Community Hub](https://community.ed-fi.org)
```